jobs/build/merge_ocp/Jenkinsfile

```diff
@@ -119,8 +119,8 @@ node {
 
                 git reset --hard HEAD
                 git clean -f -d
-                git checkout master
```

Comment:
These two steps are only trying to bring a cached copy of the git repos into a consistent state before each run. Acting on the master branch here is fine.

Reply:
alright ;D then i'll close this one in favor of https://github.com/openshift/aos-cd-jobs/pull/1973 since the the remaining relevant changes are also present there :wink: